file.c

```diff
@@ -6537,6 +6537,142 @@ const char ruby_null_device[] =
+ *  - ::grpowned?:: Returns whether the effective group of the current process
+ *                  owns the file at the given path.
+ *  - ::identical?:: Returns whether the files at two given paths are identical.
+ *  - ::owned?:: Returns whether the effective user of the current process
+ *               owns the file at the given path.
+ *  - ::pipe?:: Returns whether the file at the given path is a pipe.
+ *  - ::readable?:: Returns whether the file at the given path is readable
+ *                  by the effective user and group of the current process.
+ *  - ::readable_real?:: Returns whether the file at the given path is readable
+ *                       by the real user and group of the current process.
+ *  - ::setgid?:: Returns whether the setgid bit is set for the file at the given path.
+ *  - ::setuid?:: Returns whether the setuid bit is set for the file at the given path.
+ *  - ::socket?:: Returns whether the file at the given path is a socket.
+ *  - ::stat:: Returns the File::Stat object for the file at the given path.
+ *  - ::sticky?:: Returns whether the file at the given path has its sticky bit set.
+ *  - ::umask:: Returns the umask value for the current process.
+ *  - ::world_readable?:: Returns whether the file at the given path is readable
+ *                        by others.
+ *  - ::world_writable?:: Returns whether the file at the given path is writable
+ *                        by others.
+ *  - ::writable?:: Returns whether the file at the given path is writable by the current process.
+ *  - ::writable?:: Returns whether the file at the given path is writable
+ *                  by the effective user and group of the current process.
+ *  - ::writable_real?:: Returns whether the file at the given path is writable
+ *                       by the real user and group of the current process.
+ *
+ *  _Contents_
+ *
+ *  - ::empty? (aliasd as ::zero?):: Returns whether the file at the given path
+ *                                   exists and is empty.
+ *  - ::size:: Returns the size (bytes) of the file at the given path.
+ *  - ::size?:: Returns +nil+ if there is no file at the given path,
+ *              or if that file is empty; otherwise returns the file size (bytes).
+ *  - #size:: Returns the size (bytes) of +self+.
+ *
+ *  === Converting
+ *
+ *  - ::join:: Returns the string formed by joining the given array of strings.
+ *  - ::split:: Returns an array of two strings: the directory name and basename
+ *              of the file at the given path.
+ *
+ *  === Settings
+ *
+ *  - ::chmod:: Changes permissions of the file at the given path.
+ *  - ::chown:: Change ownership of the file at the given path.
+ *  - ::rename:: Moves the file at one given path to another given path.
+ *  - ::utime:: Sets the access time and modification time of each file
+ *              at the given paths.
+ *  - #flock:: Locks or unlocks +self+.
+ *
+ *  === Symbolic Links
```

Comment:
I don't think `Symbolic Links` should be a separate section.

* `lchmod`/`lchown`/`lutime` can move to Settings
* `link`/`symlink` can move to Creating
* `lstat`/`symlink?` can move to Types
* `readlink` can move to Paths